editor/editor_node.cpp

```diff
@@ -6102,6 +6141,13 @@ EditorNode::EditorNode() {
 	FileAccess::set_file_close_fail_notify_callback(_file_access_close_error_notify);
 
 	waiting_for_first_scan = true;
+
+	_dimming = false;
+	_dim_time = 0.0f;
+	_dim_timer = memnew(Timer);
+	_dim_timer->set_wait_time(0.01666f);
```

Comment:
Well yeah, it stops somewhere between `transition_time - 0.0166` and `transition_time`. So we a have a margin of error of one frame here.
I think that's fine for this use-case, it won't make any noticeable visual difference imo.
Unless there's an easy way to make this more precise? I'm all ears ^^